ddl/ddl_worker.go

```diff
@@ -528,6 +528,8 @@ func skipWriteBinlog(job *model.Job) bool {
 	// ActionUpdateTiFlashReplicaStatus is a TiDB internal DDL,
 	// it's used to update table's TiFlash replica available status.
 	case model.ActionUpdateTiFlashReplicaStatus:
+	// It is done without modifying table info, bin log is not needed
+	case model.ActionAlterTableAlterPartition:
```

Comment:
`switch case` won't fall through. `ActionUpdateTiFlashReplicaStatus` will return false, which is extremely serious.

Reply:
Oh... Wrote too much c/rust recently. Fixed